test/conversions.jl

```diff
@@ -35,8 +35,6 @@ module TestConversions
     @test isa(ai, Matrix{Int})
     @test ai == convert(Matrix{Int}, dt)
 
-    dt[1,1] = Nullable()
```

Comment:
Is this tested somewhere else? If not, should be kept (after adapting of course).